api/test/e2enew/upstream/upstream_test.go

```diff
@@ -826,3 +827,158 @@ var _ = ginkgo.Describe("Upstream update use patch method", func() {
 		})
 	})
 })
+
+var _ = ginkgo.Describe("test upstream delete (route is using)", func() {
```

Comment:
The description is not so good, `route is using` could be `route is in use`.

Reply:
Good advice. Let me fix it.

Reply:
Fixed.